src/connectdlg.cpp

```diff
@@ -251,6 +243,15 @@ void CConnectDlg::hideEvent ( QHideEvent* )
 void CConnectDlg::OnCentServAddrTypeChanged ( int iTypeIdx )
 {
     // store the new directory server address type and request new list
+    if ( iTypeIdx >= AT_CUSTOM )
```

Comment:
And again, just a little more comment here to say how this works - it's pretty clear but writing it out in clear English to say what it's _intended_ to do lets people confirm that's what it's doing :).